### PR TITLE
Add instructions for troubleshooting upgrade

### PR DESCRIPTION
**Pull request by fedorov**

### CODE REVIEW

File: docs/migration-guide.rst [hunk 1]
```diff
@@ -8,6 +8,14 @@ between major versions of Girder. Major version bumps contain breaking changes
 to the Girder core library, which are enumerated in this guide along with
 instructions on how to update your plugin code to work in the newer version.
 
+Existing installation may be upgraded by running ``pip install -U girder`` and
```
Comment (zachmullen) [inline docs/migration-guide.rst]: installations*

File: docs/migration-guide.rst [hunk 2]
```diff
@@ -8,6 +8,14 @@ between major versions of Girder. Major version bumps contain breaking changes
...
+re-running ``girder-install web``. You may need to remove ``node_modules`` directory
+from the installed girder package if you encounter problems while re-running
+``girder-install web``. Note that the prerequisites may have changed in the latest
+version: make sure to review
+`Prerequisites <http://girder.readthedocs.io/en/latest/prerequisites.html>`_
```
Comment (zachmullen) [inline docs/migration-guide.rst]: You can replace this line with just 

    :doc:`prerequisites`

**Comment by zachmullen** [comment]
Not sure why CircleCI isn't running this, I'll reopen the branch against origin and then merge it. Thanks!

**Comment by fedorov** [comment]
Thank you Zach!

**Pull request closed without merging**